symphony-bdk-core-invokers/symphony-bdk-core-invoker-jersey2/src/main/java/com/symphony/bdk/core/api/invoker/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -0,0 +1,171 @@
+package com.symphony.bdk.core.api.invoker.jersey2;
+
+import com.symphony.bdk.core.api.invoker.ApiClient;
+import com.symphony.bdk.core.api.invoker.ApiClientBuilder;
+import org.glassfish.jersey.SslConfigurator;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.ClientProperties;
+import org.glassfish.jersey.client.HttpUrlConnectorProvider;
+import org.glassfish.jersey.jackson.JacksonFeature;
+import org.glassfish.jersey.media.multipart.MultiPartFeature;
+
+import javax.net.ssl.SSLContext;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+/**
+ * Specific implementation of {@link ApiClientBuilder} which creates a new instance of an {@link ApiClientJersey2}.
+ */
+public class ApiClientBuilderJersey2 implements ApiClientBuilder {
+
+  private String basePath;
+  private byte[] keyStoreBytes;
+  private String keyStorePassword;
+  private byte[] trustStoreBytes;
+  private String trustStorePassword;
+  private Map<String, String> defaultHeaders;
+  private int connectionTimeout;
+  private int readTimeout;
+  private String temporaryFolderPath;
+
+  public ApiClientBuilderJersey2() {
+    basePath = "https://acme.symphony.com";
+    keyStoreBytes = null;
+    keyStorePassword = null;
+    trustStoreBytes = null;
+    trustStorePassword = null;
+    defaultHeaders = new HashMap<>();
+    connectionTimeout = 0;
+    readTimeout = 0;
```

Comment:
Looks risky to set those default values at `0`: if not overridden thought the builder it might create some confusion with weird behaviour. Just found : https://support.chargebee.com/support/discussions/topics/40956 that could help to set those default values.